bo/preisblatt.go

```diff
@@ -12,12 +12,12 @@ import (
 // verschiedenen Anwendungsfälle der Preisblätter dar.
 type Preisblatt struct {
 	Geschaeftsobjekt
-	Bezeichnung     string                  `json:"bezeichnung" validate:"required"`           // Rechnungstitel ist die Bezeichnung für die vorliegende Rechnung.
-	Sparte          sparte.Sparte           `json:"sparte,omitempty" validate:"required"`      // Preisblatt gilt für angegebene Sparte.
-	Preisstatus     preisstatus.Preisstatus `json:"preisstatus,omitempty" validate:"required"` // Merkmal, das anzeigt, ob es sich um vorläufige oder endgültige Preise handelt
-	Herausgeber     Marktteilnehmer         `json:"herausgeber" validate:"required"`           // Der Marktteilnehmer, der die Preise veröffentlicht hat. Details zum Marktteilnehmer
-	Gueltigkeit     com.Zeitraum            `json:"gueltigkeit" validate:"required"`           // Der Zeitraum für den der Preis festgelegt ist
-	Preispositionen []com.Preisposition     `json:"preispositionen" validate:"required"`       // Die einzelnen Positionen, die mit dem Preisblatt abgerechnet werden können. Z.B. Arbeitspreis, Grundpreis etc
+	Bezeichnung     string                   `json:"bezeichnung" validate:"required"`           // Rechnungstitel ist die Bezeichnung für die vorliegende Rechnung.
+	Sparte          *sparte.Sparte           `json:"sparte,omitempty" validate:"required"`      // Preisblatt gilt für angegebene Sparte.
+	Preisstatus     *preisstatus.Preisstatus `json:"preisstatus,omitempty" validate:"required"` // Merkmal, das anzeigt, ob es sich um vorläufige oder endgültige Preise handelt
```

Comment:
why are we using here the pointers?

Reply:
They're nullable in the .NET package, so we're just making the Go package consistent.